dev-docs/DEV_ENVIRONMENT.md

```diff
@@ -78,27 +78,27 @@ password, respectively.
 
 ## Setup
 
-To be able to build and run A2 on your workstation, you need the
+To be able to build and run Automate on your workstation, you need the
 following prerequisites:
 
-1. Clone A2 repo locally
+1. Clone Automate repo locally
 
    From a terminal: run `git clone https://github.com/chef/automate.git`
 
-1. Install `direnv` and integrate it into your shell
+2. Install `direnv` and integrate it into your shell
 
-   a. First, install the package. On Homebrew: run `brew install direnv`, on Ubuntu: run `apt-get install direnv`
-   b. Next, [hook direnv into your shell](https://github.com/direnv/direnv#setup)
-   c. Finally, run `direnv allow` from the A2 source directory.
+   1. First, install the package. On Homebrew: run `brew install direnv`, on Ubuntu: run `apt-get install direnv`
+   2. Next, [hook direnv into your shell](https://github.com/direnv/direnv#setup)
+   3. Finally, run `direnv allow` from the Automate source directory.
```

Comment:
Using abc's for ordered lists is not valid markdown syntax, so some of this formatting was being eaten when the document was rendered.